src/tests/ftest/util/dmg_utils.py

```diff
@@ -384,9 +400,13 @@ def storage_scan_nvme_health(self):
         """
         return self._get_result(("storage", "scan"), nvme_health=True)
 
-    def storage_query_usage(self):
+    def storage_query_usage(self, host_list=None):
```

Comment:
Not sure if we need this based upon previous comments.

Reply:
Removed.